dspace-api/src/main/java/org/dspace/authorize/service/AuthorizeService.java

```diff
@@ -449,4 +449,11 @@ public void authorizeAction(Context c, EPerson e, DSpaceObject o, int action, bo
 	void switchPoliciesAction(Context context, DSpaceObject dso, int fromAction, int toAction)
 			throws SQLException, AuthorizeException;
 
```

Comment:
Could you please add JavaDoc to service methods? What should they do, what attributes do they get?